packages/cli-platform-apple/package.json

```diff
@@ -8,6 +8,7 @@
   },
   "dependencies": {
     "@react-native-community/cli-tools": "14.0.0-alpha.0",
+    "@react-native/core-cli-utils": "nightly",
```

Comment:
Same as for Android PR, this needs to be updated once we have the first RC out

Reply:
@cortinico could you please add me, @szymonrybczak and @TMisiukiewicz as Codeowners for `@react-native/core-cli-utils` and other former CLI folders? This would make it easier for us to track changes and participate in discussions and development of these

Reply:
We don't use CODEOWNERS for react-native sadly